### PR TITLE
Changed transaction file extension from .tx to .txn

### PR DESCRIPTION
**Pull request by man-of-code**

### CODE REVIEW

**Review by amityadav0 — COMMENTED** [review]
I think in discord Robert mentioned that we should be able to load any type file (.txt or .txn)
this should be independent of that

**Comment by man-of-code** [comment]
> I think in discord Robert mentioned that we should be able to load any type file (.txt or .txn)
> this should be independent of that

Yes. You are right. But this PR is different. 
Robert mentioned that the command `sign-multisg` should accept any extension for the input transaction file. This PR is just changing the already existing functions which deal with `.tx` files.

**Comment by amityadav0** [comment]
Function you have updated in this PR is sending transaction to network, please update it to use any file extension. Let's discuss on discord if you have questions

File: examples/signed-txn/README.md [hunk 1]
```diff
@@ -14,17 +14,16 @@ Please follow the [setup](../README.md) instructions to install dependencies and
...
-goal clerk send -a <amount> -f <from_account> -t <to_account> -o out.tx -d ~/<data_directory>
+goal clerk send -a <amount> -f <from_account> -t <to_account> -o out.txn -d ~/<data_directory>
 ```
 Sign transaction (signed by `<from_account>`)
 ```bash
 goal clerk sign -i [input file] -o [output file] -d ~/<data_directory>
 ```
 
 You need to ***save the signed transaction file in `examples/multisig/assets` directory***.
- The file must have ***.tx*** extension.
```
Comment (ratik21) [inline examples/signed-txn/README.md:27]: Let's add a note here, that extension doesn't necessarily have to be *.txn*. Only that file should have a valid signed tx.

File: packages/algob/src/lib/tx.ts [hunk 2]
```diff
@@ -182,7 +182,7 @@ export async function executeTransaction (
  * firstValid and lastValid which might not be equal to the
  * current network's blockchain block height.
  * @param deployer Deployer
- * @param fileName raw signed txn .tx file
+ * @param fileName raw signed txn file
```
Comment (ratik21) [inline packages/algob/src/lib/tx.ts:185]:
```suggestion
 * @param fileName raw(encoded) signed txn file
```

**Review by ratik21 — APPROVED** [review]
Please check comments before merging.

**Pull request merged**